packages/graphql/src/schema/validation/validate-document.ts

```diff
@@ -80,9 +85,10 @@ function filterDocument(document: DocumentNode): DocumentNode {
         return fields?.filter((f) => {
             const type = getArgumentType(f.type);
```

Comment:
**nitpick (blocking)**: Should `type` be `any` here? It feels like `getArgumentType` should return `string` (Noting this due to reviewdog)